auraed/src/runtime/cells/cell.rs

```diff
@@ -172,14 +180,27 @@ pub(crate) enum CellError {
     ExecutableExists { cell_name: CellName, executable_name: ExecutableName },
     #[error("cell '{cell_name} could not find executable '{executable_name}'")]
     ExecutableNotFound { cell_name: CellName, executable_name: ExecutableName },
-    #[error("cell '{cell_name}': {source}")]
-    ExecutableError { cell_name: CellName, source: ExecutableError },
+    #[error("cell '{cell_name}' failed to start executable '{executable_name}' ({command:?}) due to: {source}")]
+    FailedToStartExecutable {
```

Comment:
should we take the Executable directly instead of all the fields?

Reply:
If we do option 5 (Insert middle) from the comment above, we could do that. As it is right now, we don't have ownership of `Executable` when this error occurs.